src/pages/RealmExplorer.js

```diff
@@ -43,24 +43,25 @@ const TYPE_MAPPINGS = {
   [TYPES.OBJECT]: [REALM_TYPES.OBJECT],
 };
 
-const typeMapper = new Map(
-  Object.entries(TYPE_MAPPINGS)
-    .map(([type, realmTypes]) => realmTypes.map(realmType => [realmType, type]))
-    .flat()
-);
+const typeMapper = new Map();
+
+Object.entries(TYPE_MAPPINGS).forEach(([type, realmTypes]) => {
+  realmTypes.forEach(realmType => {
+    typeMapper.set(realmType, type);
+  });
+});
 
 const parseType = realmType => {
   const { type } = realmType;
   if (type) return parseType(type);
   return typeMapper.get(realmType) || TYPES.OBJECT;
 };
 
-const getRealmObjects = ({ schema: objectSchemas }) =>
-  objectSchemas.map(({ schema: objectSchema }) => objectSchema.name);
+const getRealmObjects = ({ schema: objectSchemas }) => objectSchemas.map(({ name }) => name);
 
 const getRealmObjectsFields = ({ schema: objectSchemas }) =>
   objectSchemas
-    .map(({ schema: objectSchema }) => {
+    .map(objectSchema => {
       const { name, properties } = objectSchema;
```

Comment:
`schema` keyword needs to be attached to schema value while loading Realm schema. That resolves an error and runs ok in debugger mode in vscode.

`.map(({ schema: objectSchema }) => {`